hphp/runtime/vm/jit/vtune/ittnotify_config.h

```diff
@@ -90,7 +90,8 @@
 #  if ITT_PLATFORM==ITT_PLATFORM_WIN
 #    define CDECL __cdecl
 #  else /* ITT_PLATFORM==ITT_PLATFORM_WIN */
-#    if defined _M_X64 || defined _M_AMD64 || defined __x86_64__
+#    if defined _M_X64 || defined _M_AMD64 || defined __x86_64_ \
```

Comment:
This accidentally dropped the trailing `_` on `__x86_64__`, causing it to try to put the attribute on things even though it does nothing (which generates warnings). I have D3864097 up internally to fix this.